news_items/029_non_active/index.html

```diff
@@ -0,0 +1,79 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ .code-block {
+     width: 60%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+ .code-block div {
+     padding: 0.5rem;
+     display: block;
+     margin-bottom: -1em;
+     line-height: 125%;
+ }
+</style>
+
+<p>
+  Thanks to great work by 
+  <a href="https://github.com/echau01">Eugene Chau</a>, a student in
+  the University of British Columbia, it is now possible to allow
+  students to see assessments that are not available for submission,
+  either before their release date, or after their deadline.
+</p>
+
+<p>
+  This feature allows students to visualize and plan for future
+  assessments, potentially from the start of the term. It also allows
+  instructors to release completed assessments for review while
+  blocking any further submission.
+</p>
+
+<img class="border rounded" src="non-active-exam.png" width="900px">
+
+<p>
+  You can set up your assessments to be visible, but not active
```

Comment:
```suggestion
  You can set up your assessments to be visible, but not active,
```

Reply:
I decided to reword some parts of this paragraph. Can you please check it?

Reply:
Looks good! I'll add one more comma 😄